tests/com.oracle.truffle.llvm.tests.sulong/c/longdouble-div.c

```diff
@@ -0,0 +1,31 @@
+#define _GNU_SOURCE
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <math.h>
+
+void printfp(const char* msg, long double* x)
```

Comment:
Shouldn't these test cases be included in a `.include` file? Are they actually executed?

Reply:
There is no include/exclude for Sulongsuite

Reply:
Yes, Daniel has told me in person.